packages/@lwc/state/src/index.ts

```diff
@@ -224,6 +225,18 @@ export const defineState: DefineState = <
         // we have something concrete to work with, write a test, and then work backwards
       }
 
+      [disconnectContext](componentId: ContextRuntimeAdapter<object>['component']) {
+        const unsubArray = this.contextUnsubscribes.get(componentId);
+
+        if (!unsubArray || unsubArray.length === 0) {
```

Comment:
In what conditions would `length === 0`? Is this just a defensive-coding precaution, or are there real scenarios where this would be the case?

Reply:
changed it, I don't think we need a reference to `unsub` after we call it